### PR TITLE
Removes two instances of nonlatin characters in name/desc text

### LINKED ISSUE #22609: Papier-Mch Robe spelt wrong
Papier-Mch Robe spelt wrong

### LINKED ISSUE #22651: non-latin characters clearly render fine, but can't be used in descriptions?
Here is an item with non-latin characters (`é`)

```DM
/obj/item/clothing/head/canada
	name = "striped red tophat"
	desc = " It feels sticky, like maple syrup - <i>il se sent collante, comme le sirop d'érable</i>"
	icon_state = "canada"
	item_state = "canada"
```

Here it is when examined
```
That's a striped red tophat.
It feels sticky, like maple syrup - il se sent collante, comme le sirop d'Ã©rable
It is a normal-sized item.
Justin Trudeau exclaims, "Préparez-vous à mourir infidèle!"
```

So what do we have to do to make non-latin characters appear properly in descriptions?

Related #22609

### PR DESCRIPTION
**Pull request by Mervill**
fixes #22651
fixes #22609

### CODE REVIEW

**Comment by xxalpha** [comment]
Those are Latin.

**Comment by Mervill** [comment]
nonlatin in the Microsoft Code Page sense of the term

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 /obj/item/clothing/head/canada
 	name = "striped red tophat"
-	desc = " It feels sticky, like maple syrup - <i>il se sent collante, comme le sirop d'érable</i>"
+	desc = "it smells like fresh Timbits / <i>il sent comme Timbits frais</i>"
```
Comment (Supermichael777) [inline code/modules/clothing/head/misc.dm]: trademark

Comment (Mervill) [inline code/modules/clothing/head/misc.dm]: sacré bleu

Comment (Mervill) [inline code/modules/clothing/head/misc.dm]: in my defense, finding a sentence that works in both English and french _and_ also has no accents in the french isn't as easy as you might think

File: code/modules/clothing/head/misc.dm [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
 /obj/item/clothing/head/canada
 	name = "striped red tophat"
-	desc = " It feels sticky, like maple syrup - <i>il se sent collante, comme le sirop d'érable</i>"
+	desc = "it smells like fresh donut holes / <i>il sent comme des trous de beignets frais</i>"
```
Comment (Jordie0608) [inline code/modules/clothing/head/misc.dm]: What is the smell of absence of a donut center?

Comment (Mervill) [inline code/modules/clothing/head/misc.dm]: "donut hole" is the not-trademarked way to say "Timbit" aka [these](https://41.media.tumblr.com/fee7c975646d599e5f3a915ade70a77e/tumblr_inline_o47nywagGM1rr9dpz_540.jpg). They are a bite-sized fried dough confectionery popular in my country. Also the sentence translates to french without having to use an accent character.

Comment (Jordie0608) [inline code/modules/clothing/head/misc.dm]: 0/10 Tsukkomi followthrough

**Pull request merged**